copter/source/docs/autotune.rst

```diff
@@ -126,6 +159,38 @@ ATUN (auto tune overview) and ATDE (auto tune details) messages are
 written to the dataflash logs. Some details of the contents of those
 messages can be found on the :ref:`Downloading and Analyzing Data Logs in Mission Planner <common-downloading-and-analyzing-data-logs-in-mission-planner_message_details_copter_specific>` wiki page.
 
+Ground Control Station Messages
+===============================
+
+For each axis there are several phases to the tune. Rate PIDs are adjusted first, then ANGLE parameters. Progress messages during these phases are sent to the GCS (and recorded in the Dataflash logs).
+
+Typical sequence during tuning might be:
+
+::
+
+ 09:09:33	AutoTune: Twitch
```

Comment:
we can probably remove the time part of this message I guess.. I think that's probably the ground station printing that..

Reply:
copied from DataFlash log directly

Reply:
Ah, ok.. i just thought it was a bit distracting to have the time there but I'm not too fussed.